local-sns/Dockerfile

```diff
@@ -1,4 +1,4 @@
-from registry.service.opg.digital/opg-golang-alpine
+from registry.service.opg.digital/opg-golang-alpine:0.0.318
```

Comment:
Alpine not 1604 so should not be affected?

Reply:
Removed 👍